src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -54,6 +55,23 @@ def get_attrs(self, item_list, user, **kwargs):
 
             result[alert_rules[rule_activity.alert_rule.id]].update({"created_by": user})
 
+        resolved_actors = {}
+        owners_by_type = defaultdict(list)
+        for item in item_list:
+            if item.owner_id is not None:
+                owners_by_type[actor_type_to_string(item.owner.type)].append(item.owner_id)
+
+        for k, v in ACTOR_TYPES.items():
+            resolved_actors[k] = {
+                a.actor_id: a.id
+                for a in actor_type_to_class(v).objects.filter(actor_id__in=owners_by_type[k])
+            }
+
+        for alert_rule in alert_rules.values():
+            if alert_rule.owner_id:
+                type = actor_type_to_string(alert_rule.owner.type)
+                result[alert_rule]["owner"] = f"{type}:{resolved_actors[type][alert_rule.owner_id]}"
```

Comment:
We might consider making a bulk function on the `Actor` to do this at some point, but I think we should just get this pr out.

Reply:
Agreed. Will note this for a future PR.